client/QueryCompletionInformation/QueryCompletionInformationExample/QueryCompletionInformationExample.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

All other projects are net472
```suggestion
    <TargetFramework>net472</TargetFramework>
```